src/types/misc.ts

```diff
@@ -17,3 +18,25 @@ export type CompactSignature = {
   r: Hex
   yParityAndS: Hex
 }
+
+/** A key-value mapping of slot and storage values (supposedly 32 bytes each) */
+export type StateMapping = {
+  [slots: Hex]: Hex
+}
+
+export type RawAccountStateOverride = Partial<{
+  /** Fake balance to set for the account before executing the call. <32 bytes */
+  balance: Hex
+  /** Fake nonce to set for the account before executing the call. <8 bytes */
+  nonce: Hex
+  /** Fake EVM bytecode to inject into the account before executing the call. */
+  code: Hex
+  /** Fake key-value mapping to override all slots in the account storage before executing the call. */
+  state: StateMapping
+  /** Fake key-value mapping to override individual slots in the account storage before executing the call. */
+  stateDiff: StateMapping
+}>
```

Comment:
Better to be explicit about it.

```suggestion
export type RawAccountStateOverride = {
  /** Fake balance to set for the account before executing the call. <32 bytes */
  balance?: Hex
  /** Fake nonce to set for the account before executing the call. <8 bytes */
  nonce?: Hex
  /** Fake EVM bytecode to inject into the account before executing the call. */
  code?: Hex
  /** Fake key-value mapping to override all slots in the account storage before executing the call. */
  state?: StateMapping
  /** Fake key-value mapping to override individual slots in the account storage before executing the call. */
  stateDiff?: StateMapping
}
```